pkg/services/ngalert/schedule/schedule.go

```diff
@@ -209,12 +209,29 @@ type readyToRunItem struct {
 }
 
 func (sch *schedule) updateRulesMetrics(alertRules []*ngmodels.AlertRule) {
-	orgs := make(map[int64]int64)
+	liveRulesPerOrg := make(map[int64]int64)
+	pausedRulesPerOrg := make(map[int64]int64)
 	for _, rule := range alertRules {
-		orgs[rule.OrgID]++
+		if rule.IsPaused {
+			pausedRulesPerOrg[rule.OrgID]++
+			continue
+		}
+		liveRulesPerOrg[rule.OrgID]++
 	}
-	for org, numRules := range orgs {
-		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(org)).Set(float64(numRules))
+	for orgID, numRules := range liveRulesPerOrg {
+		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(orgID), "live").Set(float64(numRules))
+		if v, exists := pausedRulesPerOrg[orgID]; !exists {
+			sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(orgID), "paused").Set(0)
+		} else {
+			sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(orgID), "paused").Set(float64(v))
+		}
+	}
+	for orgID, numRules := range pausedRulesPerOrg {
+		if _, exists := liveRulesPerOrg[orgID]; exists {
+			continue
+		}
+		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(orgID), "paused").Set(float64(numRules))
+		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(orgID), "live").Set(0)
```

Comment:
I agree with Alex - I don't think we should bother resetting values. 

The one thing I have on Alex's comment is that the use `Reset()` on the vector actually makes it worst as you can get a scrape in between the reset and the computation making it seem like you just dropped all your rules.

Also, correct me if I'm wrong but resetting everything doesn't even gain us that as we're not guaranteed to receive the same `orgID` if it has no rules - effectively losing that label.

For now, I think we should leave it at:

```golang
	orgs := make(map[int64]int, len(alertRules))
	orgsPaused := make(map[int64]int, len(alertRules))
	for _, rule := range alertRules {
		orgs[rule.OrgID]++

		if rule.IsPaused {
			orgsPaused[rule.OrgID]++
		}
	}

	for org, rules := range orgs {
		paused, ok := orgsPaused[org]
        !ok { //TODO: Do we even need this?
          paused = 0
        }
		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(org), "active").Set(float64(rules - paused))
		sch.metrics.GroupRules.WithLabelValues(fmt.Sprint(org), "paused").Set(float64(paused))
	}
```

And file a bug to remove/reset scheduler metrics for an organization if it doesn't have any rules scheduled - because then, with the proper plumbing we can simply do:

```golang
	for _, deletedOrg := range deletedOrgs {
		ok := sch.metrics.GroupRules.DeleteLabelValues(fmt.Sprint(org))
	}

```